pkg/proxy/userspace/rlimit_notwindows.go

```diff
@@ -0,0 +1,25 @@
+// +build linux darwin freebsd openbsd
```

Comment:
Generally we would call this `rlimit.go` and have windows be `rlimit_windows.go`